README.md

```diff
@@ -1,28 +1,36 @@
-# Campaign Manager 360 Audience Lists Manager
-
-This repository contains Google Apps Script code that uses the
-[Campaign Manager 360 API](https://developers.google.com/doubleclick-advertisers/getting_started)
-to create and manage Audience (a.k.a remarketing) lists in bulk as well as share
-created audience lists with multiple advertisers within the same CM360 network.
-
-**DISCLAIMER**: The code samples shared here are _not_ formally supported
-by Google and are provided only as a reference. See [LICENSE](LICENSE.md)
-for more information.
-
-Contributions are highly encouraged; see [CONTRIBUTING.md](CONTRIBUTING.md).
-
-## Configuration and Setup
-
-1.  Make a copy of this
-    [template](https://docs.google.com/spreadsheets/d/1LtEflhLygH3dJ2aBFQfCZXF39b5V1KuE2FM6JlzZCG8)
-    Google Sheets spreadsheet
-1.  Create an *Apps Script* project that is linked to this spreadsheet by
-    navigating to *Tools > Script Editor*. Alternatively open the script
-    directly from the
-    [Apps Script Project Dashboard](https://script.google.com/home/all).
-1.  Copy the code samples provided in [resources/](resources) to the script
-    project created above. Consider using
-    [clasp](https://github.com/google/clasp) for easier code management.
-1.  Follow the instructions in the *Usage Guide* tab of the spreadsheet to start
-    using the solution
+<!--
```

Comment:
We don't need a specific license header for README.md files

Reply:
Shall I remove it or can it stay in there because no harm done?